tests/test_dojo.py

```diff
@@ -64,5 +64,33 @@ def test_staff_is_added_sucessfully(self):
         '''
         self.dojo.create_room('blue', 'office')
         self.assertEqual(len(self.dojo.staff), 0)
-        self.dojo.add_person('sonia', 'Staff', 'N')
+        self.dojo.add_person('kobby', 'Staff', 'N')
         self.assertEqual(len(self.dojo.all_rooms), 1, 'staff sucessfully added')
+
+    def test_print_room(self):
+        '''test that members of a room are printed'''
+        self.dojo.create_room('blue', 'office')
+        self.dojo.create_room('brown', 'livingspace')
+        self.dojo.add_person('sonia', 'fellow')
+        self.dojo.add_person('kobby', 'staff')
+        self.dojo.add_person('bett', 'fellow')
+        self.dojo.print_room('blue')
```

Comment:
Yea this test isn't testing anything. The point of a unit test is to test that the unit is doing what it's supposed to be doing. This simply calls print_room but doesn't assert that it's producing anything. I second Okerosi's suggestion, have it return a string or something and compare that to what you expect.